frontend/main.js

```diff
@@ -49,6 +49,9 @@ console.info('CONTRACTS_VERSION:', process.env.CONTRACTS_VERSION)
 console.info('LIGHTWEIGHT_CLIENT:', process.env.LIGHTWEIGHT_CLIENT)
 console.info('NODE_ENV:', process.env.NODE_ENV)
 
+// this needs to be done early
```

Comment:
AFAICT, this is the only significant change in this entire PR. I'll preemptively approve this PR, but can you please re-write the comment to also explain _why_ this needs to be done early?